BetterTeams/src/main/java/com/booksaw/betterTeams/Team.java

```diff
@@ -950,6 +950,15 @@ public void deleteTeamHome() {
 		if (useTeamHomeAsAnchor) setAnchored(false);
 	}
 
+	/**
+	 * Used to get the list of all banned players
+	 *
+	 * @return The list of UUID of all banned players
+	 */
+	public Set<UUID> getBannedPlayers() {
```

Comment:
Its probably better to make the return type of this the BanSetComponent rather than just the banned players list. 
That would allow people to use any other methods within BanSetComponent they need i.e. give them access to the `add` method to add a player to the ban list, or `save` to save an updated ban list.

You can just add the `@Getter` annotation to the variable declaration like the other Component classes